### ISSUE TITLE
Please check ioBroker.vis-justgage with js-controller 2.0

### ISSUE THREAD

**Issue by ioBrokerChecker** [iobroker-community-adapters/ioBroker.vis-justgage#24]
Hi,

the new js-controller 2.0 will come into latest repository in the next days and we want to make sure that all adapters are working well. We already did a 2 weeks Beta test and so some adapter were aleady checked and some needed slight adjustments.

You can find more information in https://github.com/ioBroker/ioBroker.js-controller/issues/482 and in the ioBroker Forum. If you have more technical questions please write in the referenced issue or in the Developer thread please. General questions are best in the genral thread.

Please update your systems to js-controller 2.0 and check your adapter.

Please close this issue once you have checked your adapter or received successfull reports from users.

Thank you very much for your support. Please contact us in the other Threads or Forum on any question.

**Comment by RustyThePropellerHead** [comment]
After the update to js-controller 2.0 the min_oid and max_oid are no longer working all my gages are 0-100 despite the max min being set (e.g. min 10 max 75). What is strange, in the editor the changes are visualised until the next webpage refresh; Furthermore, if I connect IDs from IO-Broker for max / min then the gauge is properly scaled in the runtime.

I believe it lays in the definition of max min:
```
var min = parseFloat(vis.states[data.min_oid + '.val'] || data.min_oid) || 0;
var max = parseFloat(vis.states[data.max_oid + '.val'] || data.max_oid) || 100;
```
Any thoughts?

**Comment by RustyThePropellerHead** [comment]
I have found a work-around, due to the min/max_oid's being retained in new/old varables:
1. Removed all my min/max values
2. Entered max value with a "." place...  75.0
3. And at last the min value also with a decimal place holder... 10.0 

As JavaScript isn't my thing, I haven't been able to pinpoint the issue justgage.js
It appears to me that this issue lays in the parseFloat and isNaN operations and some how the decimal place is forcing max / min to be properly set.

Cheers,
Rusty
